sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/autorest.md

```diff
@@ -56,3 +56,19 @@ directive:
   where: $.parameters.Endpoint
   transform: $["format"] = "url"
 ```
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["paths"]["/analyze-conversations/jobs"]["post"]
+      transform: >
+          $["operationId"] = "ConversationAnalysis_SubmitJob";
+```
+
+```yaml
+directive:
+    - from: swagger-document
+      where: $["paths"]["/analyze-conversations/jobs/{jobId}"]["get"]
+      transform: >
+          $["operationId"] = "ConversationAnalysis_JobStatus";
+```
```

Comment:
1. Just combine these into a list of directives. Easier to read, IMO.
````suggestion
```yaml
directive:
    - from: swagger-document
      where: $["paths"]["/analyze-conversations/jobs"]["post"]
      transform: >
          $["operationId"] = "ConversationAnalysis_SubmitJob";
          
    - from: swagger-document
      where: $["paths"]["/analyze-conversations/jobs/{jobId}"]["get"]
      transform: >
          $["operationId"] = "ConversationAnalysis_JobStatus";
```
````
2. Rather than these more complex transforms, you can use `rename-operation`: https://github.com/Azure/autorest/blob/66c3f70a8ad9ba1c5c8fdedd0378c921c06a32ce/docs/generate/built-in-directives.md#rename-operation
3. Why not just do this in the swagger itself? If this is what's right, we'd want this for all languages. /cc @mshaban-msft